sequencer/api/catchup.toml

```diff
@@ -80,4 +80,27 @@ This Endpoint fetches a leaf chain which decides a specified block height.  This
 For catching up the stake table, where `:height` is the block height of the epoch root you want to catchup to
 
 Returns a list of leaves which includes `:height` as the last leaf and should prove the block with `:height` was decided.  
-"""
\ No newline at end of file
+"""
+
+[route.reward_account]
+PATH = ["/:height/:view/reward-account/:address"]
```

Comment:
Why do we require a height and a view number to query rewards? In the implementation it looks like the height isn't used except for logging. It would seem more natural to me if we could look up reward balances by height and not view number.

Reply:
ah sorry missed that this is the catchup API